extra/3RD_PARTY_QUICK_START.md

```diff
@@ -9,48 +9,55 @@ Take a look at some of the real-life examples first:
 - https://github.com/highlightjs/highlightjs-cypher
 - https://github.com/highlightjs/highlightjs-robots-txt
 
-Basically:
+- Checkout a clone of the core repo [highlight-js](https://github.com/highlightjs/highlightjs) from GitHub.
+- Read [Language contributors checklist](https://highlightjs.readthedocs.io/en/latest/language-contribution.html).
 
-- Checkout highlight-js from github...
-- 3rd party languages are placed into the `extra` directory
+## Create a repo
 
-So if you had a `xyz` language you'd create an `extra/xyz` folder, and that would be your language module. All paths below are relative to that.
+Each language is developed in its own repo. This helps keep language definitions and maintainers independent of the core work.
+Determine if you will host the repository yourself or you want it to be part of the [highlightjs organization on GitHub](https://github.com/highlightjs).
 
-- Put your language file in `src/languages/name.js`.
-- Add detect tests in `test/detect/`.
-- Add markup tests in `test/markup/`.
-- Perhaps add a `package.json` for Node.
-- Add a nice `README`.
-- Don't forget to add a `LICENSE`.
+> To include your new language in the highlightjs organization, [create an issue](https://github.com/highlightjs/highlight.js/issues/new/choose) using the language request template and provide a description of your language and your intent to host it. We will follow up in that issue.
 
+Set your repo directory structure to follow exactly the example(s) above.
+
+For example, if you have a `language` language, create your repo directory structure as follows:
+
+- Put your language file in `src/languages/language.js`.
+- Add detect tests in `test/detect/language` (rename `language` to match your language name).
+- Add markup tests in `test/markup/language`.
+- Add a `package.json` file.
+- Include a license.
```

Comment:
```suggestion
For example, if you your grammar is named `pascal` you would create your repo directory structure as follows:

- Put your grammar file in `src/languages/pascal.js`.
- Add detect tests in `test/detect/pascal/`. 
- Add markup tests in `test/markup/pascal/`.
- Add a `package.json` file.
- Add a LICENSE so people know how your work is licensed.
```

Reply:
I agree with your suggestion, but as you also mentioned i think we should use a non-language file here because I wanted it to be the exact file used in the template project, and then very obvious all the files you need to rename. Should we go back to `xyz`?

Reply:
`xyz` is ok I think, though a placeholder is easy enough to search/replace...